src/Serilog.Enrichers.Process/project.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The process enricher for Serilog.",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "process", "enricher" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-extension-nuget.png",
+  "dependencies": {
+    "Serilog": { "target": "project" }
+  },
+  "compilationOptions": {
+    "keyFile": "../../assets/Serilog.snk"
+  },
+  "frameworks": {
+    "net45": {
+    },
+    "dotnet5.1": {
+      "dependencies": {
+        "System.Diagnostics.Process": "4.1.0-beta-23409"
+      }
+    },
+    "dotnet5.4": {
```

Comment:
Since there's no conditional compilation in this package, perhaps we can target 5.1 only?